tracee-rules/signatures/rego/regosig/traceerego.go

```diff
@@ -31,10 +31,10 @@ const querySelectedEvents string = "data.main.tracee_selected_events"
 const queryMetadata string = "data.main.__rego_metadoc__"
 
 // NewRegoSignature creates a new RegoSignature with the provided rego code string
-func NewRegoSignature(regoCode string) (types.Signature, error) {
+func NewRegoSignature(regoCode string, regoHelpersCode string) (types.Signature, error) {
```

Comment:
in the future we should make this a variadic function. doesn't have to be in this PR tough, this is a note to self

Reply:
tracked in #537